.rubocop.yml

```diff
@@ -5,6 +5,8 @@ AllCops:
   RunRailsCops: true
 Metrics/LineLength:
   Enabled: false
+Metrics/ClassLength:
+  Enabled: false
```

Comment:
Hmm, I'm actually a fan of this one. Open to ignoring Rubocop failures for this PR if you don't want to be sidetracked with refactoring a particular class when adding a single line to a class.